src/main/java/org/icgc/argo/program_service/model/entity/Cancer.java

```diff
@@ -0,0 +1,45 @@
+package org.icgc.argo.program_service.model.entity;
+
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+import org.icgc.argo.program_service.model.enums.JavaFields;
+import org.icgc.argo.program_service.model.enums.SqlFields;
+import org.icgc.argo.program_service.model.enums.Tables;
+import org.icgc.argo.program_service.model.join.ProgramCancer;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Set;
+import java.util.UUID;
+
+import static com.google.common.collect.Sets.newHashSet;
+
+
+@Entity
+@Table(name = Tables.CANCER)
+@Builder
+@Data
+@EqualsAndHashCode(of = "name")
+@AllArgsConstructor
+@NoArgsConstructor
+public class Cancer implements NameableEntity<UUID> {
+
+  @Id
+  @Column(name = SqlFields.ID, updatable = false, nullable = false)
```

Comment:
As described in this [article](https://thoughts-on-java.org/hibernate-tips-whats-the-difference-between-column-nullable-false-and-notnull/) `updatable=false` and `nullable=false` is useless, since we are not generating a schema from the annotated java entities. This should be removed for all `@Column` annotations. The only important one is the `name` parameter